README.md

```diff
@@ -47,4 +45,53 @@ Refer to the following link for more details: [System Requirements](https://help
 
 ## License
 
-Syncfusion has no liability for any damage or consequence that may arise from using or viewing the samples. The samples are for demonstrative purposes. If you choose to use or access the samples, you agree to not hold Syncfusion liable, in any form, for any damage related to use, for accessing, or viewing the samples. By accessing, viewing, or seeing the samples, you acknowledge and agree Syncfusion’s samples will not allow you seek injunctive relief in any form for any claim related to the sample. If you do not agree to this, do not view, access, utilize, or otherwise do anything with Syncfusion’s samples.
\ No newline at end of file
+Syncfusion has no liability for any damage or consequence that may arise from using or viewing the samples. The samples are for demonstrative purposes. If you choose to use or access the samples, you agree to not hold Syncfusion liable, in any form, for any damage related to use, for accessing, or viewing the samples. By accessing, viewing, or seeing the samples, you acknowledge and agree Syncfusion’s samples will not allow you seek injunctive relief in any form for any claim related to the sample. If you do not agree to this, do not view, access, utilize, or otherwise do anything with Syncfusion’s samples.
+
+## Features and Benefits
+
+### Ranges
+Radial range is a visual element that helps to quickly visualize a value where it falls on the axis.
+
+### Pointers 
+[Pointer](https://help.syncfusion.com/maui/radial-gauge/pointers?utm_source=github&utm_medium=listing&utm_campaign=maui-radial-gauge-github-samples) is used to indicate values on an axis. Radial gauge has three types of pointers: needle pointer, shape pointer, content pointer and range pointer. All the pointers can be customized as needed.
+
+### Pointer animation 
+Animates the pointer in a visually appealing way when the pointer moves from one value to another.
```

Comment:
You can interlink animates using the below link:-
https://help.syncfusion.com/maui/radial-gauge/animation

Reply:
updated